### PR TITLE
Aspell on texfiles in root directory

### PR DESCRIPTION
**Pull request by reuterk**

### CODE REVIEW

**Comment by reuterk** [comment]
Seems to hit the page limit for final.pdf now for some reason. 
Could we merge and fix the page limit in main?

**Comment by fangohr** [comment]
Great - thank you! (Was that all? I am positively surprised that only 9 files contained typos.)

**Comment by fangohr** [comment]
Just seen the 'root directory' comment.

File: concept.tex [hunk 1]
```diff
@@ -151,7 +156,7 @@ \subsubsection{Reproducibility}\label{sec:concept}\label{sec:reproducibility}
 
 Such tables and figures in a publication may be computed from the analysis of
 some type of raw data which could originate from an experiment, another publication,
-a data base, post-processing of another dataset or from executing computer
+a data base, post-processing of another data set or from executing computer
```
Comment (minrk) [inline concept.tex:159]: Single-word dataset is more common and preferred to 'data set' nowadays, I think.

Comment (reuterk) [reply concept.tex:159]: Yeah, I tried to look this up after aspell was suggesting "data set".  I had the impression that the two words are more formal and technical.
https://english.stackexchange.com/questions/2120/which-is-correct-dataset-or-data-set

**Pull request merged**